STEPS_API.md

```diff
@@ -0,0 +1,255 @@
+  - `name` - application name;
+  - `tnt_version` - version of Tarantool required by application;
+  - `package_type` - type of package (`rpm` or `deb`).
+
+### configure_instance
+
+Configure instance in runtime and change instance config.
+
+Input facts (set by role):
+- `instance_info` - information for a current instance. It's a dictionary with fields:
+  - `console_sock` - path to control socket of instance;
+  - `app_conf_file` - path to file with application config;
+  - `conf_section` - name of section in instance config file;
+  - `conf_file` - path to file with instance config;
+  - `instance_code_dir` - path to directory with distribution files.
+
+Input facts (set by config):
+- `config` - instance configuration;
+- `restarted` - if instance should be restarted or not (user forced decision);
+- `expelled` - indicates if instance must be expelled from topology;
+- `stateboard` - indicates that the instance is a stateboard;
+- `cartridge_app_name` - application name;
+- `cartridge_cluster_cookie` - cluster cookie for all cluster instances;
+- `cartridge_defaults` - default configuration parameters values for instances.
+
+Output facts:
+- `needs_restart` - if instance should be restarted to apply code or configuration changes.
+
+### restart_instance
+
+Restart instance if it should be restarted.
+
+Input facts (set by role):
+- `needs_restart` - if instance should be restarted to apply code or configuration changes;
+- `instance_info` - information for a current instance. It's a dictionary with fields:
+  - `systemd_service` - systemd service name of instance.
+
+Input facts (set by config):
+- `restarted` - if instance should be restarted or not (user forced decision).
+
+### wait_instance_started
+
+Wait until instance is fully started.
+
+Input facts (set by role):
+- `instance_info` - information for a current instance. It's a dictionary with fields:
+  - `console_sock` - path to control socket of instance.
+
+Input facts (set by config):
+- `expelled` - indicates if instance must be expelled from topology;
+- `stateboard` - indicates that the instance is a stateboard;
+- `instance_start_timeout` - time in seconds to wait for instance to be started;
+- `cartridge_wait_buckets_discovery` - indicates if routers should wait for buckets discovery after vshard bootstrap.
+
+### connect_to_membership
+
+Connect instance to membership.
+
+Input facts (set by role):
+- `instance_info` - information for a current instance. It's a dictionary with fields:
+  - `console_sock` - path to control socket of instance.
+
+Input facts (set by config):
+- `expelled` - indicates if instance must be expelled from topology;
+- `stateboard` - indicates that the instance is a stateboard;
+- `config` - instance configuration. It's a dictionary with fields:
+  - `advertise_uri` - instance URI.
+
+### set_control_instance
+
+Select instance joined to replicaset or connected to membership.
```

Comment:
Find some instance that is already in cluster. If there is no such instance, use one of instances that should be joined.

Reply:
I think we should say that user can force using some special instance as a control by setting `control_instance` variable manually.